pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}};
+        border-color: #{{page.button_background_color}};
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}};
+      {% endif %}
+    }
+
+    .microsite-content a#primary-cta {
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}};
+      {% endif %}
+    }
+
+    /* TODO: Add hover color to model or dynamically generate */
+    #primary-cta:hover {
+      opacity: 0.9;
+      transition: 0.25s;
+    }
+
+    #primary_content {
+      background-size: cover;
+      {% if page.primary_content_background_color %}
+        background-color: #{{ page.primary_content_background_color }};
+      {% endif %}
+
+      {% if page.primary_content_text_color %}
+        color: #{{ page.primary_content_text_color }};
+      {% endif %}
+
+      {% if page.primary_content_background_image %}
+        {% image page.primary_content_background_image original as primary_img %}
+        background-image: url({{ primary_img.url }});
+      {% endif %}
+    }
+
+    #primary_content a {
+      {% if page.primary_content_text_color %}
+        color: #{{ page.primary_content_text_color }};
+      {% endif %}
+    }
+
+    #secondary_content {
+      background-size: cover;
+      {% if page.secondary_content_background_color %}
+        background-color: #{{ page.secondary_content_background_color }};
+      {% endif %}
+
+      {% if page.secondary_content_text_color %}
+        color: #{{ page.secondary_content_text_color }};
+      {% endif %}
+
+      {% if page.secondary_content_background_image %}
+        {% image page.secondary_content_background_image original as secondary_img %}
+        background-image: url({{ secondary_img.url }});
+      {% endif %}
+    }
+
+    #secondary_content a {
+      {% if page.secondary_content_text_color %}
+        color: #{{ page.secondary_content_text_color }};
+      {% endif %}
+    }
+
+    #footer, .microsite .footer {
+      background-size: cover;
+      {% if page.custom_footer_background_color %}
+        background-color: #{{ page.custom_footer_background_color }};
+      {% endif %}
+
+      {% if page.custom_footer_text_color %}
+        color: #{{ page.custom_footer_text_color }};
+      {% endif %}
+
+      {% if page.custom_footer_background_image %}
+        {% image page.custom_footer_background_image original as secondary_img %}
+        background-image: url({{ secondary_img.url }});
+      {% endif %}
+    }
+
+    #footer a, .microsite .footer a {
+      {% if page.custom_footer_text_color %}
+        color: #{{ page.custom_footer_text_color }};
+      {% endif %}
+    }
+
+    .microsite-content .can_button, .microsite-content #can_embed_form input[type="submit"], .microsite-content #can_embed_form .button, .microsite-content #donate_auto_modal input[type="submit"], .microsite-content #donate_auto_modal .button {
```

Comment:
Suggest add line breaks to be more readable.

Reply:
Is there a donate modal on this page?

Reply:
Remove extra selectors not needed here? I don't see donate modal in page.

Reply:
Perhaps not for this AN form, but I just copied AN styles so the button would be changed where it would need to be changed depending on future configurations.

Reply:
Ok then suggest to add code comments here, otherwise seems confusing to have selectors that don't appear in markup.